.circleci/config.yml

```diff
@@ -262,6 +269,44 @@ defaults:
 # -----------------------------------------------------------------------------------------------
 jobs:
 
+  b_win:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - run: systeminfo
+      - run:
+          name: "Installing dependencies"
+          command: .\.circleci\win_install_dependencies.ps1
+      - run:
+          name: "Building Solidity"
+          command: .\scripts\ci\build_windows.ps1 Debug
```

Comment:
Do we build Debug in any other system? Isn't it RelWithDebug info (the default one)?

Reply:
if `CMAKE_BUILD_TYPE` is not specified at all, none of the specializations is used (no `-O0 -g` or `-O2` for example). I think it's okay to build in both modes (Release and Debug), so both compilation passes are covered by the CI.

Reply:
To be honest, I don't think it makes any sense to build in Debug mode.